seqr/views/utils/file_utils_tests.py

```diff
@@ -96,15 +106,50 @@ def test_temp_file_upload(self):
             'parsedData': PARSED_DATA,
             'uploadedFileId': mock.ANY,
         })
+        # Remove temp file
+        load_uploaded_file(response_json['uploadedFileId'])
+
+        # Test uploading with csv data
+        response = self.client.post('{}?parsedData=true'.format(url),
```

Comment:
could you test the different data types in a loop, the way `test_parse_file` does

Reply:
Okay